src/pages/SignIn/VisitorInformation.js

```diff
@@ -39,6 +41,20 @@ const useStyles = makeStyles(theme => ({
 
 const VisitorInformation = ({ form: { values } }) => {
   const classes = useStyles();
+  const [chaperones, setChaperones] = useState([]);
+  const loading = chaperones.length === 0;
```

Comment:
Normally I feel like it'd be better to use state here to set loading to false after the request succeeds, but since we don't really have error states, and this is a hackathon, I'm good with this for now lol